src/views/SelectionInfoView.tsx

```diff
@@ -51,9 +51,35 @@ export default function SelectionInfoView(
         else angleOutput = <p>Unknown angle (missing marker)</p>;
     }
     else angleOutput = <></>; //no output: not enough selected
+    /* Always display force metadata when a force file is loaded */
+    const forcesMetadata = <>
+        {
+            /* Forces, when present, are always at index 0 and 1 */
+            [0,1].map(idx => {
+                const force = forceData?.frames[frame]?.forces[idx];
+                return <p key={"f"+(idx+1)}>
```

Comment:
I think `idx` should work fine as a key directly since this is inside a fragment

Reply:
though idx isn't really meaningful here anyway, maybe just drop key and line-disable the lint